.github/workflows/build-linux-installer-deb.yml

```diff
@@ -295,3 +295,71 @@ jobs:
     - name: Remove working files to exclude from cache
       run: |
         rm -rf ./chia-blockchain-gui/packages/gui/daemon
+
+  test:
+    name: Test ${{ matrix.distribution.name }} ${{ matrix.mode.name }} ${{ matrix.arch.name }}
+    runs-on: ${{ matrix.os.runs-on[matrix.arch.matrix] }}
+    needs:
+    - build
+    container: ${{ matrix.distribution.url }}
+    strategy:
+      fail-fast: false
+      max-parallel: 4
```

Comment:
Only question is if this is really necessary. If we have the spare runners, seems like we'd want to just run as many of these at once as possible to reduce the overall time this matrix takes to run. If we're low on runners, jobs are going to be waiting either way, so might as well still just get these in the queue?